src/functions/schema.ts

```diff
@@ -94,4 +95,53 @@ export interface NonProjectTime {
   id: number;
   name: string;
   is_internal_time: boolean;
+}
+
+/**
+ * Interface for Daily Message Data
+ */
+export interface DailyMessageData {
+  message: string;
+  name: string;
+  displayDate?: string;
+  displayLogged: string;
+  displayExpected: string;
+  displayProject: string;
+  displayInternal: string;
+  billableHoursPercentage: string;
+}
+
+/**
+ * Interface for handleTimeConversion
+ */
+export interface HandleTimeConversion {
+  displayLogged: string;
+  displayExpected: string;
+  displayDifference: string;
+  displayProject: string;
+  displayInternal: string;
+}
+
+/**
+ * Interface for calculateWorkedTimeAndBillableHours
+ */
+export interface CalculateWorkedTimeAndBillableHours {
```

Comment:
Interface names in general should describe what it contains instead of where it is used. This should be changed to something like: CalculateWorkedTimeAndBillableHoursResponse